### ISSUE TITLE
"NativeMessagingInterface has stopped working"

### ISSUE THREAD

**Issue by WozardGit** [pendo324/EssentialNowPlaying#28]
Hello, thanks for this great tool!

I formatted my PC and are now experiencing issues when trying to use Google Play or SoundCloud.

When use the "Now Playing Companion" and press start while having "Essential Now Playing" open, I get this: "NativeMessagingInterface has stopped working"

The program works fine with Winamp, and "NativeMessagingInterface.exe" is in the same folder as "Essential Now Playing.exe" with "settings.json" and "manifest.json"

I've tried to delete everything related to Essential Now Playing, including the Companion, but no success.

What could the issue be?

**Comment by slimeyz** [comment]
Having the same issue, after allowing access to file URL's in the extension settings, this error popped up. But this was the only way to allow SoundCloud to pick up on the song information.

**Comment by hanmun** [comment]
Having the same issue when trying to use with Google Play.

**Comment by pendo324** [comment]
Sorry it took me so long to respond, but @hanmun just brought this issue to my attention again and we were able to resolve it by following these steps (as you can see in #40):

> 1. Close the Essential Now Playing desktop application
> 2. Open up task manager and end the "NativeMessagingInterface" process
> 3. Try running Essential Now Playing again and using the Google Play option

> Let me know if you have any questions or if some of my instructions don't make sense. Thanks!

I plan on updating the application so that this isn't necessary, but that will have to wait until this weekend.

**Comment by hanmun** [comment]
@pendo324 @slimeyz @WozardGit Yes, manually ending the NativeMessagingInterface.exe process did it for me. Also, make sure that you have the extension up and running on the source you want before re-opening the desktop app or you'll just get the same error and you'll have to do it again.

**Comment by Senpos** [comment]
I am not sure if my problem is related to this issue, but i noticed, when i close app and _remove_ companion from Chrome - **NativeMessagingInterface** starts to _consume a lot of processor time_. 

![Screenshot](http://i.imgur.com/VbPYqq8.jpg)
